source/extensions/filters/http/cache/http_cache.h

```diff
@@ -198,7 +197,7 @@ class LookupRequest {
   LookupResult makeLookupResult(Http::ResponseHeaderMapPtr&& response_headers,
                                 uint64_t content_length) const;
 
-  HeaderVector getVaryHeaders() const { return vary_headers_; }
+  const Http::RequestHeaderMapPtr& getVaryHeaders() const { return vary_headers_; }
```

Comment:
const Http::RequestHeaderMap&. No need to reference the unique_ptr when not transferring ownership.